src/amplitude-client.js

```diff
@@ -149,6 +150,10 @@ AmplitudeClient.prototype.init = function init(apiKey, opt_userId, opt_config, o
       opt_callback(this);
     }
   }
+
+  for (let i = 0; i < this._onInit.length; i++) {
+    this._onInit[i]();
```

Comment:
is it worth checking if this a function or something? I guess we don't really do it in when they give us callback functions

Reply:
Also might want to clear the `_onInit` queue after you finish running all the functions, in case they call `initialize` more than once for whatever reason...

Reply:
Going to punt on the type-check, but will add logic to clear the queue.